### PR TITLE
Add suffixes to typos from "ti" to "ty"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -52187,7 +52187,11 @@ tigthening->tightening
 tigthens->tightens
 tigthly->tightly
 tihkn->think
+tihng->thing
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52190]: Perhaps `ting` (the sound) too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -52242,9 +52253,15 @@ timzone->timezone
 timzones->timezones
 tinterrupts->interrupts
 tiome->time, tome,
+tiomeout->timeout
+tiomeouts->timeouts
 tipe->type, tip,
 tipical->typical
 tipically->typically
+tiple->triple, tuple,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52261]: Perhaps `tipple` too, although probably rare in written text.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -52242,9 +52253,15 @@ timzone->timezone
...
+tipled->tripled
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52262]: Also `tipped`?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -52242,9 +52253,15 @@ timzone->timezone
...
+tiples->triples, tuples,
+tipling->tripling
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52264]: Also `tipping`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -52510,6 +52550,15 @@ tranformer->transformer
 tranformers->transformers
 tranforming->transforming
 tranforms->transforms
+trangle->triangle, tangle,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52553]: `strangle` and even `wrangle`?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -52956,17 +53045,28 @@ treting->treating
 trew->true, threw,
 trewthful->truthful
 trewthfully->truthfully
+trgger->trigger
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53048]: Perhaps `tagger`, although probably uncommon?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -53094,11 +53228,16 @@ trotzkyite->trotskyist
 trotzkyites->trotskyists
 troubador->troubadour
 troubadors->troubadours
+troube->trouble
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53231]: `troupe`?

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -53203,6 +53357,7 @@ tuotiral->tutorial
 tuotirals->tutorials
 tupe->tuple, type,
 tupel->tuple
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53359]: `tupelo` and plural are probably uncommon, aren't they?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -53213,6 +53368,12 @@ ture->true, pure,
 turkoise->turquoise
 turle->turtle
 turly->truly
+turncate->truncate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53371]: `turncoat` is probably uncommon too.

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -53245,12 +53408,17 @@ Twosday->Tuesday
 twpo->two
 tye->type, tie,
 tyep->type
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53410]:
```suggestion
tyep->type, yep,
```

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:53371]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:53359]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:53048]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:52261]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**